tests/phpunit/tests/query/cacheResults.php

```diff
@@ -67,27 +57,64 @@ public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
 		);
 	}
 
-	function set_up() {
-		parent::set_up();
-		$this->cache_args  = null;
-		$this->new_request = null;
-		add_filter( 'wp_query_cache_key', array( $this, 'filter_wp_query_cache_key' ), 15, 3 );
-	}
 
 	/**
 	 * @dataProvider data_query_cache
```

Comment:
If the dataprovider is used for this test only then it should be renamed to `data_generate_cache_key`.